### PR TITLE
Adds the ability to toggle what a board can spawn using a screw driver.

### PR DESCRIPTION
**Pull request by MaxIsJoe**
Work on #4763

No prefabs currently use this functionality because there's a lot to go through so I'm just going to leave this up to someone else who knows how these should be assigned/edited and what should be deleted and merged into one board.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Systems/Construction/ComputerCircuitboard.cs [hunk 1]
```diff
@@ -6,15 +7,35 @@ namespace Items.Construction
...
 	/// </summary>
-	public class ComputerCircuitboard : MonoBehaviour
+	public class ComputerCircuitboard : MonoBehaviour, ICheckedInteractable<HandApply>
 	{
 		[Tooltip("Computer which should be spawned when this circuitboard's frame is constructed.")]
 		[SerializeField]
 		private GameObject computerToSpawn = null;
 
+		[SerializeField] private ItemTrait screwdriver;
+		[SerializeField] private List<GameObject> customTypes = new List<GameObject>();
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Systems/Construction/ComputerCircuitboard.cs:18]: so nothing uses currently? , could you hook at the script on the computer frame to use this if it set

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Systems/Construction/ComputerCircuitboard.cs:18]: ..it already does tho?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Systems/Construction/ComputerCircuitboard.cs:18]: computerToSpawn  isn't changed?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Systems/Construction/ComputerCircuitboard.cs:18]: image

you sure? :)

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Systems/Construction/ComputerCircuitboard.cs:18]: Make sure to set it via awake then , To So it So it picks up from customTypes with the current currentIndex

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Systems/Construction/ComputerCircuitboard.cs:18]: Done 👍

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**